src/interface.js

```diff
@@ -14,7 +14,8 @@ function xdateToData(xdate) {
     month: xdate.getMonth() + 1,
     day: xdate.getDate(),
     timestamp: XDate(dateString, true).getTime(),
-    dateString: dateString
+    dateString: dateString,
+    friendly: xdate.toString('dddd MMMM d')
```

Comment:
'friendly' is not a clear prop name. Also, will be better to allow the user to send the format he wants instead of setting it hard coded.